datafusion/sqllogictest/test_files/coalesce.slt

```diff
@@ -23,7 +23,7 @@ select coalesce(1, 2, 3);
 1
 
 # test with first null
-query IT
+query ?T
```

Comment:
What happened here? Does this say the type of `coalesce(null, 3, 2, 1)` is now Null? But the result of `arrow_typeof(coalesce(null, 3, 2, 1))` is still Int64 😕